### ISSUE TITLE
Dump two handed Katana, add Spade and wate rpurification device

### ISSUE THREAD

**Issue by rknntns** [danielmiessler/BugOutBag#4]
The Two handed Katana adds a lot of valuable weight. I would suggest a Cold Steel SF Spade instead. With sharpened edges it could be used as a weapon as well as a cutting tool, plus you could dig with it. Digging can be really valuable out in the wild since you can dig a shallow trench around your tent, protecting against heavy rain, plus you also could use it as an axe.
I would also recommend some device to purify water, either a water filtration system or purification tablets

**Comment by coreyjjohnson** [comment]
I must agree. You have a nice list but I'd replace the katana with an axe, tomahawk, or spade. As much as I love my axes, the spade is probably better. More useful, still can chop, still a weapon.
